rewrite-java/src/main/java/org/openrewrite/java/JavaPrinter.java

```diff
@@ -848,6 +848,17 @@ public J visitVariableDeclarations(VariableDeclarations multiVariable, PrintOutp
             p.append(']');
         }
         if (multiVariable.getVarargs() != null) {
+            if (!(multiVariable.getTypeExpression() instanceof ArrayType)) {
+                // The `visit(multiVariable.getTypeExpression(), p);` statement above does not know the
```

Comment:
Perhaps these comments can go, as removing this exception would currently cause the build to fail

Reply:
It's very rare for us to throw exceptions; can this be reworked not to throw?